src/protocols/kubernetes/url.c

```diff
@@ -109,26 +110,42 @@ int guac_kubernetes_endpoint_attach(char* buffer, int length,
                 sizeof(escaped_pod), kubernetes_pod))
         return 1;
 
-    /* Generate attachment endpoint URL */
-    if (kubernetes_container != NULL) {
+    /* Generate endpoint path depending on the call type */
+    char* call="attach";
+    if (use_exec)
+        call = "exec";
+
+    char endpoint_path[GUAC_KUBERNETES_MAX_ENDPOINT_LENGTH];
+    snprintf(endpoint_path, GUAC_KUBERNETES_MAX_ENDPOINT_LENGTH*3, 
+    "/api/v1/namespaces/%s/pods/%s/%s", escaped_namespace, escaped_pod, call);
+
+    /* Generate endpoint params */
+    char endpoint_params[GUAC_KUBERNETES_MAX_ENDPOINT_LENGTH]="";
+    int param_length=0;
+
+    if(use_exec){
+        /* Escape exec command */
+        if (guac_kubernetes_escape_url_component(escaped_exec_command,
+                    sizeof(escaped_exec_command), exec_command))
+            return 1;
 
+        param_length += snprintf(endpoint_params, GUAC_KUBERNETES_MAX_ENDPOINT_LENGTH, 
+        "command=%s&", escaped_exec_command);
```

Comment:
As above, when breaking a statement across multiple lines, please indent subsequent lines such that this is clear.